UI/RemoteISOScreen.cpp

```diff
@@ -397,7 +398,7 @@ void RemoteISOScreen::CreateViews() {
 		rightColumnItems->Add(new Choice(sy->T("Share Games (Server)")))->OnClick.Handle(this, &RemoteISOScreen::HandleStartServer);
 		browseChoice->SetEnabled(true);
 	}
-	Choice *settingsChoice = new Choice(sy->T("Settings"));
+	Choice *settingsChoice = new Choice(pa->T("Settings"));
```

Comment:
Yes indeed "Pause" has nothing to do here...
See my reply to your other comment below for an idea of a better category.